server/Dockerfile

```diff
@@ -37,8 +48,11 @@ USER 1000
 COPY docker-entrypoint.sh /usr/local/bin
 COPY is_healthy.sh /usr/local/bin
 COPY is_running.sh /usr/local/bin
+# S2I Scripts
+COPY .s2i /usr/local/s2i
 
-ENTRYPOINT ["docker-entrypoint.sh"]
+# OpenShift S2I requires to use CMD not ENTRYPOINT here
+CMD ["docker-entrypoint.sh"]
```

Comment:
This breaks the image, it does not allow to pass in properties anymore. Example:
```
$ docker run -it jboss/infinispan-server -Djboss.default.jgroups.stack=tcp
docker: Error response from daemon: oci runtime error: container_linux.go:247: starting container 
process caused "exec: \"-Djboss.default.jgroups.stack=tcp\": executable file not found in $PATH".
```

Is this bug still not fixed on openshift? I see openshift/source-to-image#475 was closed